### PR TITLE
Updated UG

### PR DESCRIPTION
**Pull request by ljiazh3ng**
clear points 3.0 for #478

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -57,26 +57,37 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
-* `..` (Move to parent note)
-* `exampleNote` (Move to child node titled "exampleNote")
+The `RelativePath` operates similarly to the command line input that you may be familiar with.
+Listed below are the notations used to define a `RelativePath`.
 
-[NOTE]
-The title of a Note can take in alphanumeric variables, symbols (except ` and -) and spaces.
+* `.` (Point to current note)
+* `..` (Point to parent note)
+* `exampleNote` (Point to child node titled "exampleNote")
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: "Point to a child note" instead?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -57,26 +57,37 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+
+**Validity of Paths**
+
+- AbsolutePath
+** Must start with the root note.
+** After every `/` you can provide a `Title` <<TitleValidity, (Valid Title)>>
+
+- Relative Path
+** Starts with `.` or `..` or a Title <<TitleValidity, (Valid Title)>>
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: Do you think it's useful to add an example below each of these to clarify what you mean by these definitions?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -57,26 +57,37 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
 **Example**
 
 The picture below helps to illustrate the use of `RelativePath` and `AbsolutePath`
 
 .DataStructure example to illustrate Path
-image::PathExample1.png[width="790"]
+image::UGPathDiagram.png[width="790"]
+
 
 The `AbsolutePath` `/CS2101` is equivalent to `RelativePath` `../CS2101`
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: I think you mean `/Workspace/CS2101`?

**Review by kevinputera — APPROVED** [review]
Hey @ljiazh3ng , overall LGTM! I left some comments below.

One more thing! I think the info section that contains the text "We cannot open or delete the parent of the root note because it does not exist. e.g Delete /../ will result in an error." has a slight error. `/../` is not a valid path, by definition!

File: docs/UserGuide.adoc (hunk 1 above)
Comment (ljiazh3ng) [reply docs/UserGuide.adoc]: done!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (ljiazh3ng) [reply docs/UserGuide.adoc]: sure i think it's better to add a few examples.

File: docs/UserGuide.adoc (hunk 3 above)
Comment (ljiazh3ng) [reply docs/UserGuide.adoc]: Oh yes, i forgot to change it after changing the diag. Thanks for pointing it out.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -57,36 +57,49 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
 
-Thus `open /CS2101` and `open ../CS2101` would yield the same result.
+Thus `open /Workspace/CS2101` and `open ../CS2101` would yield the same result.
 
 [NOTE]
 Paths are used by the `open` and `delete` command
 
 [WARNING]
-We cannot open or delete the parent of the root note because it does not exist. e.g `Delete /../` will result in an error.
+We cannot open or delete the parent of the root note because it does not exist. e.g `Delete /` will result in an error.
```
Comment (kevinputera) [inline docs/UserGuide.adoc]: Actually, if you do `Delete /`, it'll delete the root note instead of the parent of the root note!

Comment (ljiazh3ng) [reply docs/UserGuide.adoc]: Sorry, i've made changes to it . Can you help me take a look if its valid?

**Pull request merged**